georiviere/portal/views/portal.py

```diff
@@ -17,3 +17,14 @@ class PortalViewSet(mixins.RetrieveModelMixin, viewsets.GenericViewSet):
 
     def get_queryset(self):
         return Portal.objects.all()
+
+    def get_serializer_context(self):
+        """
+        Extra context provided to the serializer class.
+        """
+        return {
+            'request': self.request,
+            'format': self.format_kwarg,
+            'view': self,
+            'portal_pk': self.kwargs['pk']
```

Comment:
get context from super then  add your values instead of manually override everything